conftest.py

```diff
@@ -103,8 +103,8 @@ def pytest_sessionstart(session):
 
     if session.config.option.collectonly:
         return
-
-    context.scenario.session_start()
+    if not hasattr(session.config, "workerinput"):
+        context.scenario.session_start()
```

Comment:
Too risky. My case works!!! ...but the other cases...

Reply:
gooooood catch !